.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+### Description
+
+<!--- Please include a summary of the change and which issue is fixed. --->
+<!--- Please also include relevant motivation and context. -->
+
+<!--- Add visual representation of the effect of the change when possible, e.g. before and after screenshots, code snippets, etc. --->
+
+Fixes #{issue}
+
+### How Has This Been Tested?
+
+<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce while reviewing your changes. -->
+
+### AI Disclosure
+<!--- Remove this section if your PR does not contain AI-generated content. --->
+- [ ] This PR contains AI-generated content.
+  - [ ] I have tested any AI-generated content in my PR.
+  - [ ] I take responsibility for any AI-generated content in my PR.
```

Comment:
I'm a little confused about this statement - I think overall, for every contribution, we are responsible for introducing changes. 
Do you think it needs a separate mention under `AI`?